resources/.ktlint_editorconfig

```diff
@@ -4,3 +4,11 @@ ktlint_standard_max-line-length = 160
 ktlint_standard = enabled
 ktlint_standard_multiline-expression-wrapping = disabled
 ktlint_standard_string-template-indent = disabled
+ktlint_standard_chain-method-continuation = disabled
+ktlint_standard_class-signature = disabled
```

Comment:
Same here. I think you're looking for:
ktlint_class_signature_rule_force_multiline_when_parameter_count_greater_or_equal_than

This is part of the Kotlin conventions also; if you have multiple parameters, the Kotlin Conventions suggest multi-line. They've just made it strict at >1.

Reply:
That is certainly a rule that might be relevant, but it's not what I am trying to avoid by disabling this.